source/_posts/2016-08-07-using-recursion-to-extract-particular-elements-from-a-list.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Using recursion to extract particular elements from a list"
+kewyords: "filter, collection, enumerable"
+category: collections
+comments: true
+---
+
+In Elixir, or Erlang or any other functionnal programming language, you can easily manipulate lists using recursion.
+In particular, you can extract elements from a list and separate them from the rest of the elements, in an efficient way.
```

Comment:
Remove 'in particular', Change elements -> 'individual elements'. Change 'efficient' -> 'concise'